.travis.yml

```diff
@@ -43,6 +43,9 @@ matrix:
         - postgresql
 
     - env:
+        # Enabling Deprecations when running tests
+        # You can set the environment variable PANDAS_TESTING_MODE to deprecate to show any DeprecationWarning message. 
+        # This is enabled in the travis builds for numpy 1.9 (and evenutally for the python 3.4 build).
```

Comment:
I think can delete this last line; pretty out of date

Reply:
Yep, also you can rephrase a bit, since this is a comment and not in the docs anymore. Something like `PANDAS_TESTING_MODE="deprecate" causes DeprecationWarning messages to be displayed in the logs. See pandas/_testing.py for more details.`